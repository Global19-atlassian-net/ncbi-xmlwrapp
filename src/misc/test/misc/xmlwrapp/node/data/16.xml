<?xml version="1.0"?>
<root xmlns:a="one" xmlns:b="two" xmlns="default">
    <child xmlns:a="another" xmlns:b="two">
        <nested xmlns:a="one" b:attr="value">
            <a:deep xmlns:a="one" xmlns="anotherDefault"/>
        </nested>
    </child>
</root>
