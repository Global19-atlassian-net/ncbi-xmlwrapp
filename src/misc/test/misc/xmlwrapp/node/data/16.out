<?xml version="1.0"?>
<root xmlns="default">
  <child xmlns:b="two">
    <nested b:attr="value">
      <a:deep xmlns:a="one"/>
    </nested>
  </child>
</root>

